package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@digitalbazaar/bitstring",
+  "version": "0.0.1-0",
+  "description": "Bitstring module for universal JavaScript",
+  "license": "BSD-3-Clause",
+  "main": "index.js",
+  "scripts": {
+    "test": "npm run test-node",
+    "test-node": "cross-env NODE_ENV=test mocha -r esm --preserve-symlinks -t 30000 -A -R ${REPORTER:-spec} --require tests/test-mocha.js tests/*.spec.js",
+    "test-karma": "karma start karma.conf.js",
+    "coverage": "cross-env NODE_ENV=test nyc --reporter=lcov --reporter=text-summary npm run test-node",
+    "coverage-ci": "cross-env NODE_ENV=test nyc --reporter=lcovonly npm run test-node",
+    "coverage-report": "nyc report",
+    "lint": "eslint ."
+  },
+  "files": [
+    "main.js",
+    "index.js"
+  ],
+  "dependencies": {
+    "base64url-universal": "^1.1.0",
+    "esm": "^3.2.25",
+    "karma-chrome-launcher": "^3.1.0",
+    "karma-mocha": "^2.0.1",
+    "karma-webpack": "^4.0.2",
+    "mocha": "^8.0.1",
```

Comment:
These should be dev dependencies.

Reply:
Makes sense, since tests are run only in development environment.